templates/tags/list.html

```diff
@@ -5,7 +5,7 @@
         {{ super() }}
     {% endblock header %}
 
-    <h1>Tags</h1>
+    <h1>{% if config.extra.after_dark_multilingual %}{{ trans(key="tags", lang=lang) }}{% else %}Tags{% endif %}</h1>
```

Comment:
The `config.extra.after_dark_multilingual` flag is only here for backwards compatibility. If breaking existing configurations is OK, or if there is a way to check if a translation key exists, the flag can be removed.

Reply:
I believe you can just check `config.language | length`, if it's more than 1 then it's a multi-language site?